src/run/troubleshoot/analysis/AwsEnvironmentVariableAnalysis.ts

```diff
@@ -0,0 +1,92 @@
+import { execSync } from "child_process";
+import BaseAnalysis from "./AbstractBaseAnalysis.js";
+import AnalysisOutcome from "./AnalysisOutcome.js";
+import { AnalysisIssue, TroubleshootAnalysisTaskContext } from "./AnalysisTask.js";
+
+const ANALYSIS_HEADLINE = "Checks AWS environment variables";
+
+const SUGGESTIONS = [
+    "Run the below command to set the AWS profile or AWS region:",
+    "`export AWS_PROFILE=development-eu-west-2` or `export _AWS_REGION=eu-west-2`"
+];
+
+const DOCUMENTATION_LINKS = [
+    "troubleshooting-remedies/correctly-add-aws-environment-variables.md"
+];
+
+/**
+ * An analysis task that evaluates whether the required AWS environment variables are set or correctly configured.
+ */
+export default class AwsEnvironmentVariableAnalysis extends BaseAnalysis {
+
+    async analyse ({ config }: TroubleshootAnalysisTaskContext): Promise<AnalysisOutcome> {
+        const profileIssues = this.checkAwsProfileEnvVariables();
+
+        const issues = profileIssues || this.checkAwsRegionEnvVariables();
+        const level = profileIssues ? "Fail" : "Warn";
+
+        return this.createOutcomeFrom(ANALYSIS_HEADLINE, issues, level);
+    }
+
+    /**
+     * Checks the AWS_PROFILE environment variable value
+     * Ensures the value is set and is one of the available profiles
+     * @returns { Promise<AnalysisIssue | undefined> } - Returns an issue if value is unset or not one of the available profile, otherwise undefined
+     */
+    private checkAwsProfileEnvVariables (): AnalysisIssue | undefined {
+        const AWS_PROFILE = process.env.AWS_PROFILE;
+        const _AWS_PROFILE = process.env._AWS_PROFILE;
+        const _DEFAULT_AWS_PROFILE = process.env._DEFAULT_AWS_PROFILE;
```

Comment:
See former point - has no bearing on chs-dev, `AWS_PROFILE` is the only one which will mean that aws picks up, all the others are irrelevant